framework/deproxy_client.py

```diff
@@ -21,9 +22,19 @@ def __init__(self, *args, **kwargs):
         self.rps = 0
         self.valid_req_num = 0
         self.cur_req_num = 0
+                               # These parametes control heavy chunked testing
+                               # You can set it programmaticaly or via client config
+        self.segment_size = 0; # TCP segment size, bytes, 0 for disable
```

Comment:
With this setting we generate chunks of constant size, which limits the tests flexibility. https://github.com/tempesta-tech/tempesta-test/issues/115 proposes to use a generator for chunk sizes, so we can run tests with a pseudorandom chunks